operator/attestor/notifier.go

```diff
@@ -7,6 +7,12 @@ import (
 	"github.com/NethermindEth/near-sffl/operator/consumer"
 )
 
+type Predicate = func(consumer.BlockData) bool
```

Comment:
`Predicate` seems a bit too general, something like `BlockPredicate` would be better IMO.

Reply:
Also, why is this public and `subscriberData` isn't?

Reply:
`Predicate` is part of public API and `subscriberData` is only used within the file

Reply:
Alright, that's okay. It can be used outside